modules/admin_manual/pages/appliance/configuration/clamav.adoc

```diff
@@ -42,10 +42,19 @@ The installation should only take a few minutes.
 [[configure-owncloud-to-use-clamav]]
 == Configure ownCloud to Use ClamAV
 
+Start the ClamAV service:
+
+....
```

Comment:
Code blocks must be enclosed in `----` not `....`.

Reply:
all the code blocks I have seen were in dots. The minuses break the code block

Reply:
Have a look at https://github.com/owncloud/docs/edit/master/modules/admin_manual/pages/appliance/configuration/clamav.adoc, line 55 - 58. That has a good example.

Reply:
why is it different here then? https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#source-code